src/modules/displayList.js

```diff
@@ -1,10 +1,9 @@
 import retrieveFromStore from './retrieveFromStore.js';
 import selectItem from './selectItem.js';
 import deleteItem from './deleteItem.js';
-import listStore from './listStore.js';
-
 // eslint-disable-next-line import/no-cycle
```

Comment:
- [ ] Please avoid disabling linters as they are very useful to keep a consistent code format and ensure the use of best practices. Instead, please consider solving the errors! 🤓 🔧